websites/docs-smoke-test/gatsby-config.js

```diff
@@ -9,8 +9,9 @@ module.exports = {
   // https://www.gatsbyjs.com/docs/reference/release-notes/v2.30
   flags: {
     DEV_SSR: true,
-    FAST_REFRESH: true,
     PRESERVE_WEBPACK_CACHE: true,
+    FAST_DEV: true,
+    PRESERVE_FILE_DOWNLOAD_CACHE: true,
```

Comment:
Last time I tried using these there were some issues. Have you encountered any problems with these flags being active?